static/docs/get-started/initialize.md

```diff
@@ -1,12 +1,15 @@
 # Initialize
 
-In order to start using DVC, you need first to initialize it in your
-<abbr>workspace</abbr>. DVC doesn't require Git and can work without any source
-control management system, but for the best experience we recommend using DVC on
-top of Git repositories.
+First of all install DVC from `pip` (on the [installation](/doc/install) page
```

Comment:
Let's just point users to the Install page. This `pip` command is broken (it'll require `sudo`), I also don't like it because it is breaking the environment, we are not installing any cloud support (it'll be a big surprise), `pip` is def not a first choice on a lot of platforms (I'll probably change the order on other pages to make pip go after conda).

I understand the concerns here - it's better to make it self-sufficient. But on the other hand, it's really bad idea to install it this way on most platforms.